plugins/gcp/fix_plugin_gcp/resources/compute.py

```diff
@@ -967,16 +967,23 @@ def fetch_instances(group: str) -> None:
                             request_parameter_in={"project", "zone", "instanceGroup"},
                             response_path="items",
                         )
-                        path_data = urlparse(group).path
-                        zone = path_data.split("/")[5]
-                        instance_group = path_data.split("/")[7]
+                        path_data = urlparse(group).path.split("/")
+                        zone = path_data[5]
+                        instance_group = path_data[7]
```

Comment:
for example we have a group link like:
```
https://compute.googleapis.com/compute/v1/projects/{project}/zones/{zone}/instanceGroups/{instanceGroup}/listInstances
```
1. We will take `compute/v1/projects/{project}/zones/{zone}/instanceGroups/{instanceGroup}/listInstances`
2. From `compute/v1/projects/{project}/zones/{zone}/instanceGroups/{instanceGroup}/listInstances` we will take {zone} and {instanceGroup}